lib/validation.rb

```diff
@@ -173,6 +173,17 @@ def self.validate_port_range(port_range)
     end_port ? [start_port, end_port] : [start_port]
   end
 
+  def self.validate_port(port)
+    fail ValidationFailed.new({port: "Port must be between 0 to 65535"}) unless port.to_i.to_s == port.to_s
```

Comment:
The `validate_positive_int` validator and `validate_usage_limit` validator don't have the second `to_s`
```suggestion
    fail ValidationFailed.new({port: "Port must be between 0 to 65535"}) unless port.to_i.to_s == port
```